Your_second_ASDF_converter.ipynb

```diff
@@ -0,0 +1,1142 @@
+    "    and manifests.  This method is registered with the\n",
+    "    asdf.resource_mappings entry point.\n",
+    "\n",
+    "    Returns\n",
+    "    -------\n",
+    "    list of collections.abc.Mapping\n",
+    "    \"\"\"\n",
+    "    from . import resources\n",
+    "    resources_root = importlib_resources.files(resources)\n",
+    "\n",
+    "    return [\n",
+    "        DirectoryResourceMapping(\n",
+    "            resources_root / \"schemas\", \"asdf://stsci.edu/example-project/schemas/\"),\n",
+    "        DirectoryResourceMapping(\n",
+    "            resources_root / \"manifests\", \"asdf://stsci.edu/example-project/manifests/\"),\n",
+    "    ]\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2f70eef1",
+   "metadata": {},
+   "source": [
+    "The key element in this file are the last few lines. What this is \n",
+    "indicating is that when a schema id is encountered and the beginning\n",
+    "of that schema_id matches the last string, the remainder of that\n",
+    "schema id is taken from the schema ID and appended to the local\n",
+    "path for the resource directory plus the \"schemas\" subdirectory\n",
+    "to construct the path to the actual schema file.\n",
+    "\n",
+    "Likewise for the manifest, where the manifest id is located in the \n",
+    "extensions.py file for the converter package. Basically, it tells the\n",
+    "converter package, where the manifest file may be found on the filesystem,\n",
+    "and from that, where the schemas may be found on the local filesystem.\n",
+    "\n",
+    "Did we say last file? Not quite. We need an empty `__init__.py` to make\n",
+    "this a package."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d47a1b5e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%writefile src/myschemas/__init__.py\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ebb6630a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%writefile src/myschemas/resources/__init__.py\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e5b7344a",
+   "metadata": {},
+   "source": [
+    "Installing the packages\n",
+    "---------------------------"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "4a67e777",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pip install --editable ."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8261435e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cd ../myconverters"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "dd7dd553",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pip install --editable ."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "bf7e5895",
+   "metadata": {},
+   "source": [
+    "Testing the converters\n",
```

Comment:
Worth showing a validation failure in this section?

Reply:
I was thinking about that before. Sure I can do that.